### DISCUSSION TITLE
Github seems to be down?

### DISCUSSION THREAD

**Discussion by maxpaleo** [community/community#37430] [Other Feature Feedback, Questions, & Ideas] [answered]
Github seems to be down?

**Comment by radar-blip** [comment]
Working fine here. Any 500 errors?

**Comment by DenverCoder1** [comment, marked as answer]
You can check https://www.githubstatus.com.

There seems to have been a short issues / codespaces [incident earlier today](https://www.githubstatus.com/history), but everything appears operational currently.
